android/common/src/main/AndroidManifest.xml

```diff
@@ -60,7 +60,10 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_LOCATION_EXTRA_COMMANDS" />
-    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION" />
+    <uses-permission android:name="android.permission.ACTIVITY_RECOGNITION" />
+    <!-- Use the following if API target is < 29 -->
+    <uses-permission android:name="android.gms.permission.ACTIVITY_RECOGNITION"/>
```

Comment:
I would advise to test it with a production build using ionic, I think this permission require authentication on Google cloud platform